ui/packages/consul-ui/app/components/consul/kind/index.hbs

```diff
@@ -1,89 +1,99 @@
-                    Mesh gateways
-                  </a>
-                </li>
+  {{#if withInfo}}
+    <dl class="tooltip-panel">
+      <dt>
+        <span data-test-kind={{item.Kind}} class="consul-kind">
+          {{Name}}
+        </span>
+      </dt>
+      <dd>
+        <MenuPanel @position="left">
+          <BlockSlot @name="header">
+            {{#if (eq item.Kind 'ingress-gateway')}}
+              Ingress gateways enable ingress traffic from services outside the Consul service mesh to services inside the Consul service mesh.
+            {{else if (eq item.Kind 'terminating-gateway')}}
+              Terminating gateways allow connect-enabled services in Consul service mesh to communicate with services outside the service mesh.
+            {{else if (eq item.Kind 'api-gateway')}}
+              API gateways enable ingress traffic from services outside the Consul service mesh to services inside the Consul service mesh.
+            {{else}}
+              Mesh gateways enable routing of Connect traffic between different Consul datacenters.
+            {{/if}}
+          </BlockSlot>
+          <BlockSlot @name="menu">
+            <li role="separator">
+              {{#if (eq item.Kind 'ingress-gateway')}}
+                  About Ingress gateways
+              {{else if (eq item.Kind 'terminating-gateway')}}
+                  About Terminating gateways
+              {{else if (eq item.Kind 'api-gateway')}}
+                  About API gateways
+              {{else}}
+                  About Mesh gateways
               {{/if}}
-              {{#if (not-eq item.Kind 'terminating-gateway')}}
-                <li role="none" class="docs-link">
-                  <a tabindex="-1" role="menuitem" href={{concat (env 'CONSUL_DOCS_URL') (get link 'terminating-gateway')}} rel="noopener noreferrer" target="_blank">
-                    Terminating gateways
-                  </a>
-                </li>
-              {{/if}}
-              {{#if (not-eq item.Kind 'ingress-gateway')}}
-                <li role="none" class="docs-link">
-                  <a tabindex="-1" role="menuitem" href={{concat (env 'CONSUL_DOCS_URL') (get link 'ingress-gateway')}} rel="noopener noreferrer" target="_blank">
-                    Ingress gateways
-                  </a>
-                </li>
-              {{/if}}
-          {{/let}}
-              </BlockSlot>
-            </MenuPanel>
-          </dd>
-        </dl>
-      {{else}}
-          <span data-test-kind={{item.Kind}} class="consul-kind">
-            {{Name}}
-          </span>
-      {{/if}}
-  {{/let}}
+            </li>
+      {{#let (from-entries (array
+          (array 'ingress-gateway' '/consul/developer-mesh/ingress-gateways')
+          (array 'terminating-gateway' '/consul/developer-mesh/understand-terminating-gateways')
+          (array 'mesh-gateway' '/consul/developer-mesh/connect-gateways')
```

Comment:
We only have [the Learn guide covering API Gateway for Kubernetes](https://developer.hashicorp.com/consul/tutorials/developer-mesh/kubernetes-api-gateway), so I suppose it makes sense to leave API gateway out of this list for now

Reply:
Yeah, I was thinking we could add it eventually. As is it'll still show a learn guide link, but it'll just link to the root of the learn guides, which I think is fine, didn't want to change this component around too much.